app/models/tag.rb

```diff
@@ -37,7 +37,7 @@ def path
   end
 
   def run_count
-    self.count = NodeTag.joins(:node).where(tid: tid).where('node.status = 1').count
+    self.size = NodeTag.where(tid: tid).size
```

Comment:
lets leave the check for status here

` self.count = NodeTag.joins(:node).where(tid: tid).where('node.status = 1').size`